packages/shared/src/lib/localLanguage.ts

```diff
@@ -57,16 +48,14 @@ export const countryCodeToVerfiedLocalLanguage = (
 ): LocalLanguage | undefined => {
     const hasCountryLanguageOrDefault = localLanguages[countryCode] || dfltLocalLanguage;
     if (
-        testName === (LocalLanguageEpicTestName || LocalLanguageBannerTestName) &&
-        variantName === (LocalLanguageEpicVariant || LocalLanguageBannerVariant) &&
-        hasCountryLanguageOrDefault
+        testName === hasCountryLanguageOrDefault?.testName &&
+        variantName === hasCountryLanguageOrDefault?.variantName
     ) {
         return (
             localLanguages[countryCode] ?? {
+                testName: dfltLocalLanguage.testName,
+                variantName: dfltLocalLanguage.variantName,
```

Comment:
It doesn't appear that the `testName` and `variantName` are being used in the banner router so I think this function and file could be simplified

Reply:
Ok, removed